arrayfire/array.py

```diff
@@ -1185,20 +1185,35 @@ def to_list(self, row_major=False):
         ct_array, shape = self.to_ctype(row_major, True)
         return _ctype_to_lists(ct_array, len(shape) - 1, shape)
 
-    def __repr__(self):
+    def to_string(self):
```

Comment:
An obvious solution would have been to find out if PyCharm sets environment variables and disable prints if they are set.

A preliminary search says this is possible: http://stackoverflow.com/a/29782618/2192361